front/src/form/bsdd/utils/packagings.ts

```diff
@@ -14,7 +14,7 @@ export function getPackagingInfosSummary(packagingInfos: PackagingInfo[]) {
     (acc, packagingInfo) => acc + packagingInfo.quantity,
     0
   );
-  const packages = packagingInfos
+  const packages = [...packagingInfos]
     .sort((p1, p2) => p1.type.localeCompare(p2.type))
```

Comment:
Pour info c'est le `sort()` qui fait péter le code. Je sais pas pourquoi honnêtement. Si vous avez une idée, je suis curieux!

Reply:
le .sort c'est du in place, ama ça fait comme si on essayait muter le state directement